guide/onboarding/introduction.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: guide
+title: Onboarding
+description: An overview of some of the concepts to consider when building onboarding experiences.
+nav_order: 2
+has_children: true
+permalink: /guide/onboarding/introduction/
+---
+
+<!--
+
+Editor's notes
+
+A brief introduction and summary of all pages in this section. The idea is that readers
+scan this page to get an overview of the section and then decide which topics to dive into.
+
+-->
+
+# Onboarding
+
+Effective onboarding can make or break someone’s first bitcoin experience. Therefore, it is important to identify your target audience when building wallet-products and design specifically with that audience in mind. 
+
+Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard. Users might assume that the patterns and habits from prior experience will apply, so convincing them to use a different financial system might be better in the product. 
+
+On the other hand, more than four billion people have no exposure to traditional financial institutions. This, along with designing for younger people whose ideas about money aren’t fully formed, represents a very different challenge.
+
+Onboarding, for this guide, refers to anything that helps users set up the product or learn its interface (such as dedicated flows and UI elements). But onboarding isn’t just for first-time users. Experienced users also need to be onboarded, especially when new features are added. 
+
+Remember: onboarding should not be a crutch for bad design. Avoid over indexing on trying to explain too much within onboarding, and then shooting users into a poorly designed app. Rather, think about onboarding flows as a smooth on-ramp into a great app experience.
```

Comment:
```suggestion
Remember: Onboarding should not be a crutch for bad design. Avoid trying to explain too much and then shooting users into a poorly designed app. Rather, think about onboarding flows as a smooth on-ramp into a great user experience.
```

Simplified a little bit. The short paragraph used the term "onboarding" three times which felt repetitive.